specification/cosmos-db/resource-manager/Microsoft.DocumentDB/preview/2020-06-01-preview/nova.json

```diff
@@ -0,0 +1,1051 @@
+              },
+              "tokens": {
+                "type": "integer",
+                "format": "int32",
+                "description": "The number of tokens set for the node."
+              },
+              "owns": {
+                "type": "number",
+                "format": "double",
+                "description": "The percentage of the data owned by the node per datacenter times the replication factor (e.g., 33.3, or null if the data is not available). For example, a node can own 33% of the ring, but shows 100% if the replication factor is 3. For non-system keyspaces, the endpoint percentage ownership information is shown."
+              },
+              "hostId": {
+                "type": "string",
+                "description": "The network ID of the node."
+              },
+              "rack": {
+                "type": "string",
+                "description": "The rack this node is part of."
+              }
+            }
+          }
+        }
+      }
+    },
+    "ListBackups": {
+      "description": "List of restorable backups for a Cassandra cluster.",
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "Container for array of backups.",
+          "type": "array",
+          "readOnly": true,
+          "items": {
+            "type": "object",
+            "properties": {
+              "id": {
+                "type": "string",
+                "description": "Unique resource identifier for this backup, of the form /subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.DocumentDB/cassandraClusters/{clusterName}/backups/{backupId}. The backupId is the timestamp of when the backup was taken, in number of seconds since the epoch."
+              },
+              "timestamp": {
+                "type": "string",
+                "format": "date-time",
+                "description": "The time this backup was taken, formatted like 2021-01-21T"
+              }
+            }
+          }
+        }
+      }
+    },
+    "ListDataCenters": {
+      "description": "List of managed Cassandra data centers and their properties.",
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "Container for array of data centers.",
+          "type": "array",
+          "readOnly": true,
+          "items": {
+            "$ref": "#/definitions/DataCenterResource"
+          }
+        }
+      }
+    },
+    "SeedNode": {
+      "type": "object",
+      "properties": {
+        "ipAddress": {
+          "description": "IP address of this seed node.",
+          "type": "string"
+        }
+      }
+    },
+    "Certificate": {
+      "type": "object",
+      "properties": {
+        "pem": {
+          "description": "PEM formatted public key.",
+          "type": "string"
+        }
+      }
```

Comment:
I am not sure if we can include keys in a Resource body. Here is the ARM guideline [https://armwiki.azurewebsites.net/api_contracts/guidelines/rpc.html#impact-9](https://armwiki.azurewebsites.net/api_contracts/guidelines/rpc.html#impact-9)

Reply:
These are public keys the customer needs to be able to get easily. They are not secrets.